test/sql/copy/test_copy.cpp

```diff
@@ -1626,3 +1626,684 @@ TEST_CASE("Test CSV reading/writing from relations", "[relation_api]") {
 
 	REQUIRE_THROWS(con.ReadCSV(csv_file, {"i INTEGER); SELECT 42;--"}));
 }
+
+TEST_CASE("Test copy into auto from lineitem csv", "[copy]") {
```

Comment:
Perhaps move your tests to a separate file `test_auto_csv.cpp` or something? The test_copy is beginning to be pretty full :)

Reply:
Indeed, I split them up!